packages/react-charts/src/components/ChartDonutUtilization/ChartDonutUtilization.tsx

```diff
@@ -627,7 +626,7 @@ export const ChartDonutUtilization: React.FunctionComponent<ChartDonutUtilizatio
     const result = [];
     if (thresholds) {
       // Ensure thresholds are in sorted order
-      const sThresholds = orderBy(thresholds, 'value', invert ? 'desc' : 'asc');
+      const sThresholds = Array.from(thresholds).sort((a, b) => (invert ? b.value - a.value : a.value - b.value));
```

Comment:
You may be able to remove lodash from react-core, but it will still be installed via Victory.
And since lodash shall be installed, we have been using it throughout the charts package.

There are more places where lodash is used below.

![Screenshot 2024-01-11 at 11 04 25 AM](https://github.com/patternfly/patternfly-react/assets/17481322/16241782-7926-43f8-bf9d-04209a89bf36)

Reply:
I wasn't aware that Victory also depends on Lodash, so I logged an issue there as well https://github.com/FormidableLabs/victory/issues/2711. Still, considering Lodash is no longer actively maintained, it cannot harm to remove some of the code here?